lte/gateway/python/integ_tests/s1aptests/test_attach_detach_flaky_retry_success.py

```diff
@@ -0,0 +1,77 @@
+"""
+Copyright 2022 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import unittest
+
+import s1ap_types
+from flaky import flaky
+from integ_tests.s1aptests import s1ap_wrapper
+
+
+class TestAttachDetachFlakyRetrySuccess(unittest.TestCase):
+    """Integration Test: TestAttachDetachFlakyRetrySuccess"""
+
+    def setUp(self):
+        """Initialize before test case execution"""
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
+
+    def tearDown(self):
+        """Cleanup after test case execution"""
+        self._s1ap_wrapper.cleanup()
+
+    @flaky(max_runs=3, min_passes=3)
+    def test_attach_detach_flaky_retry_success(self):
+        """Basic attach/detach test for a single UE with flaky retry
+
+        This testcase runs the basic attach-detach scenario multiple times even
+        for successful case to validate TFW and S1APTester cleanup
+        """
+        num_ues = 2
+        detach_type = [
+            s1ap_types.ueDetachType_t.UE_NORMAL_DETACH.value,
+            s1ap_types.ueDetachType_t.UE_SWITCHOFF_DETACH.value,
+        ]
+        wait_for_s1 = [True, False]
+        self._s1ap_wrapper.configUEDevice(num_ues)
+
+        for i in range(num_ues):
+            req = self._s1ap_wrapper.ue_req
+            print(
+                "************************* Running End to End attach for ",
+                "UE id ",
+                req.ue_id,
+            )
+            # Now actually complete the attach
+            self._s1ap_wrapper._s1_util.attach(
+                req.ue_id,
+                s1ap_types.tfwCmd.UE_END_TO_END_ATTACH_REQUEST,
+                s1ap_types.tfwCmd.UE_ATTACH_ACCEPT_IND,
+                s1ap_types.ueAttachAccept_t,
+            )
+
+            # Wait for EMM Information from MME
+            self._s1ap_wrapper._s1_util.receive_emm_info()
+            print(
+                "************************* Running UE detach for UE id ",
+                req.ue_id,
+            )
```

Comment:
Since there are only 2 UEs I guess you can add flow verification here

Reply:
I have added the flow rule creation/deletion verification